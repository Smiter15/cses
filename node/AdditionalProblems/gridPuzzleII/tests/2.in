10
4 2 0 4 4 1 1 0 2 0
2 4 2 2 3 1 0 1 1 2
651 333 480 442 712 653 318 547 881 229
96 263 664 263 376 436 897 462 557 975
759 646 895 895 849 476 781 520 984 48
348 635 381 519 76 92 171 394 329 742
623 426 4 287 379 380 413 966 532 970
940 291 305 525 876 845 0 346 364 985
395 712 619 776 230 385 559 91 469 578
833 783 3 838 760 72 908 172 37 440
141 668 421 136 192 296 981 883 332 344
867 727 747 175 193 667 250 442 759 720
