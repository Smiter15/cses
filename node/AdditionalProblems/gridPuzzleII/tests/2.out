11747
...XXX..X.
..X......X
..........
XXXX......
XX..X....X
....X.....
.X........
..........
.X.....X..
..........
